developing/cycle/tracker.md

```diff
@@ -1,7 +1,8 @@
 # [Development Cycle](./README.md) / Pivotal Tracker
 
 ## Overview
+
 We use Pivotal Tracker to track tasks and estimate how much time each task will take.
 
-* 🔒[How to organize a Pivotal project](https://docs.google.com/document/d/1abmxQdYol6Zi3kR3U__JNJl3uPdKF9WPYWfu75AkZ50/edit#heading=h.yyb7wuihugn8)
-* [Pivotal workflow](https://www.pivotaltracker.com/help/articles/workflow_overview/)
+- 🔒[How to organize a Pivotal project](https://docs.google.com/document/d/1abmxQdYol6Zi3kR3U__JNJl3uPdKF9WPYWfu75AkZ50/edit#heading=h.yyb7wuihugn8)
```

Comment:
This PR seems to be consistently changing * to -. Why? Hard to know if this is intentional without context

Reply:
My apologies for not doing a better job of explaining context.  markdownlint complained about using `*` for bullets instead of `-` so I changed it.

Reply:
That's … odd. We use `*` on pretty much all our other docs. I'm confused as to why it was complaining about these …

Reply:
I ... I can't explain what happened, so some kind of user error must have occurred, I just don't know what.  Let me Try Again with a much smaller set of changes.

Thank you for paying attention to the details.

Reply:
Ok, here's an example: ```
developing/languages/README.md: 35: MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
```

What I misunderstood about the error is that it just means the list character needs to be consistent in the document, not that it needs to be changed everywhere.  Apologies.  Updated PR shortly.

Reply:
I've redone the PR to be less invasive and updated the PR description to hopefully better describe what is going on.

Apologies for the sloppiness.